dids/src/main/kotlin/web5/sdk/dids/methods/dht/DidDht.kt

```diff
@@ -146,27 +147,33 @@ public sealed class DidDhtApi(configuration: DidDhtConfiguration) : DidMethod<Di
     }
 
     // map to the DID object model's verification methods
-    val verificationMethods = (opts.verificationMethods?.map { (key, purposes) ->
-      VerificationMethod.builder()
-        .id(URI.create("$id#${key.keyID}"))
-        .type("JsonWebKey2020")
-        .controller(URI.create(id))
-        .publicKeyJwk(key.toPublicJWK().toJSONObject())
-        .build().also { verificationMethod ->
-          purposes.forEach { relationship ->
-            relationshipsMap.getOrPut(relationship) { mutableListOf() }.add(
-              VerificationMethod.builder().id(verificationMethod.id).build()
-            )
-          }
-        }
-    } ?: emptyList()) + identityVerificationMethod
-    opts.services?.forEach { service ->
+    val verificationMethodToPublicKey = opts.verificationMethodsToAdd
+      .toPublicKeys(keyManager)
+      .associate { (_, publicKey) ->
+        val key = publicKey.publicKeyJwk
+        val verificationMethod = VerificationMethod.builder()
+          .id(URI.create("$id#${publicKey.id}"))
```

Comment:
we need to make sure there are no ids of `0` which would conflict with the identity key

Reply:
Done